navigator_engine/app.py

```diff
@@ -27,6 +28,8 @@ def create_app(config_object=None):
     db.init_app(app)
     api.init_app(app)
 
+    importlib.import_module('navigator_engine.pluggable_logic')  # register pluggable_logic
```

Comment:
Why are you importing it like this instead of?
```
import navigator_engine.pluggable_logic
```

Reply:
It would look like I was importing something that wasn't then ever used.  Doing it this way I believe makes it more apparent that the line is actually running some important code. It's less likely to be seen as an error (my editor would otherwise underline it with a yellow squiggle saying import statement is never used - it's confusing)